app/assets/javascripts/components/group/group_create_modal.es6.jsx

```diff
@@ -36,14 +36,13 @@ class GroupCreateModal extends CreateModal {
       conference_id: this.props.conference.id,
       letter: this.state.letter,
     }
-    ConferenceActions.createGroup(params)
+    ConferenceActions.createGroup(this.props.template)
   }
 
-  generateHandler(field) {
-    var state = {};
-    return(event) => {
-      state[field] = event.target.value;
-      this.setState(state);
+  generateAction() {
```

Comment:
Just to be more general/reusable, can you make `generateAction` take in a `field` parameter.
Instead of hardcoding `letter` in we'll pass that as the parameter!
Example from `RoomCreateModal`:
```
  generateHandler(field) {
    return(event) => {
      var value = event.target.value;
      if (this.props.type === 'conference') {
        ConferenceActions.storeAttribute(field, value);
      } else if (this.props.type === 'rooms') {
        RoomsActions.storeAttribute(field, value);
      }
    };
  }
```